README.md

```diff
@@ -294,6 +294,35 @@ warning is given.
 
 ```
 
+### How to create a ceature card
+
+This can only create a creature with **power/toughness** and **no keyword** as so far. It can perform attack action.
```

Comment:
```suggestion
Pass a `power` and `toughness` to the `Creature` class:
```

What you had written is perhaps more than necessary here. Just a simple instruction is all that is needed.